package.json

```diff
@@ -43,6 +43,7 @@
   "resolutions": {
     "braces": "^3.0.3",
     "micromatch": "^4.0.8",
-    "cross-spawn": "^7.0.5"
+    "cross-spawn": "^7.0.5",
+    "brace-expansion": "^1.1.12"
```

Comment:
nit. the wanted braces-expansion version is `^1.1.7`, so `1.1.11` can be resolved without this force resolution line since it's in the same major version, but doesn't matter practically